administrator/components/com_patchtester/script.php

```diff
@@ -48,6 +51,24 @@ public function __construct()
 	 */
 	public function postflight($type, $parent)
 	{
-		$this->removeFiles();
```

Comment:
This line has to stay. It calls the function on the parent class.

Reply:
So I need to reinstate this line then in my PR?

Reply:
Correct.